trolldb/database/mongodb.py

```diff
@@ -103,39 +97,47 @@ class MongoDB:
         us, we would like to fail early!
     """
 
-    __client: Optional[AsyncIOMotorClient] = None
-    __database_config: Optional[DatabaseConfig] = None
-    __main_collection: AsyncIOMotorCollection = None
-    __main_database: AsyncIOMotorDatabase = None
+    __client: ClassVar[Optional[AsyncIOMotorClient]] = None
+    __database_config: ClassVar[Optional[DatabaseConfig]] = None
+    __main_collection: ClassVar[Optional[AsyncIOMotorCollection]] = None
+    __main_database: ClassVar[Optional[AsyncIOMotorDatabase]] = None
 
-    default_database_names = ["admin", "config", "local"]
+    default_database_names: ClassVar[list[str]] = ["admin", "config", "local"]
     """MongoDB creates these databases by default for self usage."""
 
     @classmethod
+    @validate_call
```

Comment:
This severely limits duck typing...

Reply:
Done. `@validate_call` has been now removed.